adaptive_breakpoints/test/adaptive_breakpoints_test.dart

```diff
@@ -72,8 +72,8 @@ void main() {
             : falseColor,
       )),
     );
-    var box = tester.widget(find.byType(Container));
-    expect((box as Container).color, trueColor);
+    var box = tester.widget<Container>(find.byType(Container));
```

Comment:
these <Container> annotations don't seem to be always needed

Reply:
Without this annotation, `box` would have type `Widget` instead of `Container`, for which the analyzer would complain
```
  error • The getter 'color' isn't defined for the type 'Widget' at test/adaptive_breakpoints_test.dart:23:16 • (undefined_getter)
```

Also, it looks like the annotation is only needed during declaration (`var box`).